numba/targets/imputils.py

```diff
@@ -8,6 +8,7 @@
 import functools
```

Comment:
The two comments below relating to docstrings:
https://github.com/numba/numba/pull/1599/files#r48951208
https://github.com/numba/numba/pull/1599/files#r48951240
, if considered valid, are applicable to most function definitions that are decorators within this file.